### PR TITLE
Update app's level

### PR DESCRIPTION
**Pull request by yalh76**
- [x] [agendav](https://ci-apps.yunohost.org/ci/job/5)
- [x] [airsonic](https://ci-apps.yunohost.org/ci/job/6)
- [x] [collabora](https://ci-apps.yunohost.org/ci/job/31)
- [x] [diaspora](https://ci-apps.yunohost.org/ci/job/42)
- [x] [fallback](https://ci-apps.yunohost.org/ci/job/58)
- [x] [friendica](https://ci-apps.yunohost.org/ci/job/301)
- [x] [grav](https://ci-apps.yunohost.org/ci/job/309)
- [x] [halcyon](https://ci-apps.yunohost.org/ci/job/258)
- [x] [ ihatemoney](https://ci-apps.yunohost.org/ci/job/90)
- [x] [jenkins](https://ci-apps.yunohost.org/ci/job/96)
- [x] [jirafeau](https://ci-apps.yunohost.org/ci/job/299)
- [x] [lstu](https://ci-apps.yunohost.org/ci/job/110)
- [x] [luckysheet](https://ci-apps.yunohost.org/ci/job/111)
- [x] [mautrix_whatsapp](https://ci-apps.yunohost.org/ci/job/122)
- [x] [minetest](https://ci-apps.yunohost.org/ci/job/127)
- [x] [nextcloud](https://ci-apps.yunohost.org/ci/job/145)
- [x] [osticket](https://ci-apps.yunohost.org/ci/job/154)
- [x] [ owntracks](https://ci-apps.yunohost.org/ci/job/155)
- [x] [pgadmin](https://ci-apps.yunohost.org/ci/job/160)
- [x] [radicale](https://ci-apps.yunohost.org/ci/job/178)
- [x] [rainloop](https://ci-apps.yunohost.org/ci/job/179)
- [x] [snipeit](https://ci-apps.yunohost.org/ci/job/191)
- [x] [unattended_upgrades](https://ci-apps.yunohost.org/ci/job/213)

### CODE REVIEW

**Comment by kay0u** [comment]
it looks like:
[owntracks](https://ci-apps.yunohost.org/ci/job/155)
[minetest](https://ci-apps.yunohost.org/ci/job/127)
[mautrix_whatsapp](https://ci-apps.yunohost.org/ci/job/122)
[fallback](https://ci-apps.yunohost.org/ci/job/58)
are just a regression in package_check, I propose to keep the current level

**Review by ericgaspar — APPROVED** [review]

File: apps.json [hunk 1]
```diff
@@ -1017,7 +1018,7 @@
     "friendica": {
         "branch": "master",
         "category": "social_media",
-        "level": 8,
+        "level": 4,
```
Comment (kay0u) [inline apps.json:1021]:
```suggestion
        "level": 8,
```

File: apps.json [hunk 2]
```diff
@@ -1272,7 +1273,7 @@
     "grav": {
         "branch": "master",
         "category": "publishing",
-        "level": 8,
+        "level": 2,
```
Comment (kay0u) [inline apps.json:1276]:
```suggestion
        "level": 8,
```

File: apps.json [hunk 3]
```diff
@@ -1533,7 +1535,7 @@
     "jirafeau": {
         "branch": "master",
         "category": "synchronization",
-        "level": 8,
+        "level": 0,
```
Comment (kay0u) [inline apps.json:1538]:
```suggestion
        "level": 6,
```

**Pull request merged**